h/db/__init__.py

```diff
@@ -136,6 +137,25 @@ def _maybe_create_world_group(engine, authority):
     session.close()
 
 
+def _maybe_create_world_organization(engine, authority):
+    from h import models
+    session = Session(bind=engine)
+    world_org = session.query(models.Organization).filter_by(pubid='__world__').one_or_none()
+    if world_org is None:
+        world_org = models.Organization(name=u'Hypothesis',
+                                        authority=authority,
+                                        pubid = '__world__',
+                                        )
+        with open(r'h/static/images/icons/logo.svg', 'r') as h_logo:
```

Comment:
What was the `r` prefix for here? The string doesn't contain any escape sequences. I guess that would have been necessary on Windows systems when using backslashes in paths?

Reply:
Yes. It's kinda muscle memory at this point. I 'spose it's not strictly necessary tho.